scripts/spi_flash/board_defs.py

```diff
@@ -87,6 +87,26 @@
         'spi_bus': 'spi3a',
         'cs_pin': 'PA15'
     },
+    'btt-skr-2': {
+        'mcu': "stm32f407xx",
+        'spi_bus': "swspi",
+        'spi_pins': "PC8,PD2,PC12",
+        'cs_pin': "PC11",
+        'skip_verify': True
+    },
```

Comment:
The `btt-skr-2` looks like it could be added as an alias for `btt-octopus-f407-v1`.

Reply:
Noted, will change that to be an alias